packages/tools/playground/src/tools/loadManager.ts

```diff
@@ -122,7 +140,7 @@ export class LoadManager {
                             const encodedData = payload.unicode;
                             const decoder = new TextDecoder("utf8");
 
-                            code = decoder.decode(DecodeBase64ToBinary(encodedData));
+                            code = decoder.decode((DecodeBase64ToBinary || DecodeBase64ToBinaryReproduced)(encodedData));

```

Comment:
DecodeBase64ToBinary will never be undefined ???

Reply:
of course it will :-) If you use Babylon 4.2.1 which doesn't include it.